src/datasources/products/types.ts

```diff
@@ -37,16 +37,18 @@ export const PropertiesOptions = {
   FILE_IDS: 'fileIds',
 };
 
-export const ProductPropertiesProjectionMap = {
-  'Product ID': PropertiesOptions.ID,
-  'Part number': PropertiesOptions.PART_NUMBER,
-  'Product name': PropertiesOptions.NAME,
-  'Family': PropertiesOptions.FAMILY,
-  'Updated at': PropertiesOptions.UPDATEDAT,
-  'Workspace': PropertiesOptions.WORKSPACE,
-  'Keywords': PropertiesOptions.KEYWORDS,
-  'Properties': PropertiesOptions.PROPERTIES,
-  'File IDs': PropertiesOptions.FILE_IDS
+export const productsProjectionLabelLookup: Record<Properties, {
+  label: string;
+  projection: string;}> = {
+  [Properties.id]: { label: 'Product ID', projection: PropertiesOptions.ID},
+  [Properties.partNumber]: { label: 'Part number', projection: PropertiesOptions.PART_NUMBER },
+  [Properties.name]: { label: 'Product name', projection:  PropertiesOptions.NAME },
+  [Properties.family]: { label: 'Family', projection:  PropertiesOptions.FAMILY },
+  [Properties.updatedAt]: { label: 'Updated at', projection: PropertiesOptions.UPDATEDAT },
+  [Properties.workspace]: { label: 'Workspace', projection: PropertiesOptions.WORKSPACE },
+  [Properties.keywords]: { label: 'Keywords', projection: PropertiesOptions.KEYWORDS },
+  [Properties.properties]: { label: 'Properties', projection: PropertiesOptions.PROPERTIES },
+  [Properties.fileIds]: { label: 'File IDs', projection: PropertiesOptions.FILE_IDS },
 }
```

Comment:
Since we have the same values between the enum and the constant, we could do something like


```suggestion
export const productsProjectionLabelLookup: Record<Properties, {
  label: string;
  projection: Properties;
}> = {
  [Properties.id]: { label: 'Product ID', projection: Properties.id },
  [Properties.partNumber]: { label: 'Part number', projection: Properties.partNumber },
  [Properties.name]: { label: 'Product name', projection: Properties.name },
  [Properties.family]: { label: 'Family', projection: Properties.family },
  [Properties.updatedAt]: { label: 'Updated at', projection: Properties.updatedAt },
  [Properties.workspace]: { label: 'Workspace', projection: Properties.workspace },
  [Properties.keywords]: { label: 'Keywords', projection: Properties.keywords },
  [Properties.properties]: { label: 'Properties', projection: Properties.properties },
  [Properties.fileIds]: { label: 'File IDs', projection: Properties.fileIds },
};
```

This would improve the typing